Mathlib/LinearAlgebra/LinearIndependent.lean

```diff
@@ -171,9 +184,15 @@ variable (R v) in
 theorem linearIndepOn_empty : LinearIndepOn R v ∅ :=
   linearIndependent_empty_type ..
 
-theorem linearIndependent_set_subtype {s : Set ι} :
```

Comment:
I couldn't find a `@[deprecated] alias` for this rename. Since all the other renames in this PR seem to have matching aliases, is this intentional?

Reply:
I missed this - thanks!